templates/resources/cloud_project_kube.md.tmpl

```diff
@@ -54,6 +54,7 @@ The following arguments are supported:
 * `name` - (Optional) The name of the kubernetes cluster.
 * `region` - a valid OVHcloud public cloud region ID in which the kubernetes cluster will be available. Ex.: "GRA9". Defaults to all public cloud regions. **Changing this value recreates the resource.**
 * `version` - (Optional) kubernetes version to use. Changing this value updates the resource. Defaults to the latest available.
+* `plan` - (Optional) Plan of the MKS cluster `free` or `standard`. Default to `free`. Migration to another plan is not implemented yet.
```

Comment:
could you please run `tfplugindocs` to regenerate the doc from the templates+examples ? (sorry this is not documented yet, but you have to do it to have an up-to-date doc)